lib/vanagon/platform/windows.rb

```diff
@@ -192,14 +192,14 @@ def generate_msi_package(project) # rubocop:disable Metrics/AbcSize
         #   -dr             - Root DirectoryRef to point all components to
         #   -var            - Replace "SourceDir" in the @source attributes of all components with a preprocessor variable
         app_heat_flags = " -dr INSTALLDIR -v -ke -indent 2 -cg AppComponentGroup -gg -srd -t wix/filter.xslt -sreg -var var.AppSourcePath "
-        app_heat_flags += " -fips" if project.platform.name =~ /windowsfips-2012r2/
+        app_heat_flags += " -fips" if project.platform.name =~ /windowsfips-(2012r2|2016)/
```

Comment:
To future proof this could we do something like

```suggestion
app_heat_flags += " -fips" if project.platform.is_windows? && project.platform.is_fips?
```

And since this is heat/wix which only makes sense on windows, maybe we can drop the first condition?